encoding/graph/thing/edge.rs

```diff
@@ -354,8 +332,7 @@ impl<'a> Prefixed<'a, BUFFER_KEY_INLINE> for ThingEdgeHasReverse<'a> {}
 
 impl<'a> Keyable<'a, BUFFER_KEY_INLINE> for ThingEdgeHasReverse<'a> {
     fn keyspace(&self) -> EncodingKeyspace {
-        // TODO: may be worth caching since it's not free to compute but static within an instance
-        Self::keyspace_for_from(self.from())
+        EncodingKeyspace::Data
```

Comment:
Can we keep the old one? I think I have half a vision to split LONG and SHORT attributes into differnt RocksDB Databases so we can optimise their bloom filters separately

Reply:
That'd be a larger change. `prefix_from_prefix` would be impossible, e.g., since `StorageKey` requires `Keyspace`, but we won't be able to tell it just from the attribute prefix.
I'd rather not pretend here that we have a solution.

Reply:
hmm okay fair enough if it is already partial!